packages/cxl-lumo-styles/src/templates/vaadin-details.js

```diff
@@ -0,0 +1,12 @@
+import lumoVaadinDetails from '../styles/vaadin-details-css.js';
+
+const $template = document.createElement('template');
+
+$template.innerHTML = `
+<dom-module id="cxl-lumo-details" theme-for="vaadin-details">
+  <template>
+    <style>${lumoVaadinDetails}</style>
+  </template>
+</custom-style>`;
```

Comment:
Yo, wrong closing tag? Shouldn't this be `</dom-module>`?

Same for almost everywhere else.

But I'm about done refactoring this, so maybe no immediate additional action needed.